.flake8

```diff
@@ -1,4 +1,6 @@
 [flake8]
 max-line-length = 100
+extend-select = B950
+extend-ignore = E203, E501, E701
```

Comment:
These look a redundant for the script at least. You still want them?

Reply:
How do you feel about https://github.com/wtsi-npg/npg_porch_cli/pull/4/commits/aa594ee323a880fbe29a7be12410b7cb8b54eb73 and starting from a clean slate?